aes/decryptor_test.go

```diff
@@ -0,0 +1,108 @@
+package aes
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"testing/quick"
+)
+
+func TestDecrypt(t *testing.T) {
+	expected := "foo"
+	key := strings.Repeat("f", 32)
```

Comment:
Sure, but why does this matter? I wasn't going for 0xffffffffffffffffffffffffffffffff